tests/unittests/TraceEventsTest.cpp

```diff
@@ -108,6 +108,36 @@ class TraceEventsTest : public ::testing::TestWithParam<std::string> {
     }
   }
 
+  // In the function below we're trying to compare auto instrumented trace
+  // output with optimized model graph. Since the code generation may make
+  // further changes to the graph representation we limit checking to a set of
+  // convolution / maxpool / cpumaxsplat kinds which are never changed.
+  std::set<std::string> checkedKinds = {"convoluition", "maxpool",
+                                        "cpumaxsplat"};
+
+  void checkEventMetadata(
+      const std::vector<TraceEvent> &traceEvents,
+      std::vector<std::pair<std::string, std::string>> expected) {
```

Comment:
it's test, does not matter to much, but can you make it `const vector&`

Reply:
I actually repeated the interface of the same function above. But yeah, of course, no any reason to copy this vector here.